### PR TITLE
Blueprint https://blueprints.launchpad.net/or/+spec/traindpuinfo-web-server

### PR DESCRIPTION
**Pull request by mbm-OR**
Blueprint https://blueprints.launchpad.net/or/+spec/traindpuinfo-web-server

More info: http://www.elvastower.com/forums/index.php?/topic/26249-remote-control-dpu-units/page__view__findpost__p__281433

### CODE REVIEW

File: Source/RunActivity/Viewer3D/WebServices/Web/index.html [hunk 1]
```diff
@@ -30,13 +30,14 @@
...
 	<ul>
-	  <li><a href="/TrackMonitor/index.html">Track Monitor (F4)</a></li>
-	  <li><a href="/TrainDriving/index.html">Train Driving (Ctrl+F5)</a></li>
-	  <li><a href="/TrackMonitorTrainDriving/index.html">Track Monitor and TrainDriving</a></li>
-	  <li><a href="/HUD/index.html">Head Up Display (F5)</a></li>
-	  <li><a href="/CabControls/index.html">Cab Controls</a></li>
-	  <li><a href="/Time/index.html">Time</a></li>
-	  <li><a href="/APISample/index.html">API Sample</a></li>
+		<li><a href="/TrackMonitor/index.html">Track Monitor (F4)</a></li>
+		<li><a href="/TrainDpu/index.html">Train Distribued Power (Shift+F9)</a></li>
```
Comment (cjakeman) [inline Source/RunActivity/Viewer3D/WebServices/Web/index.html:34]:
```suggestion
		<li><a href="/TrainDpu/index.html">Train Distributed Power (Shift+F9)</a></li>
```

Comment (cjakeman) [reply Source/RunActivity/Viewer3D/WebServices/Web/index.html:34]: Just a typo

Comment (mbm-OR) [reply Source/RunActivity/Viewer3D/WebServices/Web/index.html]: Thank you.

**Review by cjakeman — APPROVED** [review]
Hi Mauricio,

This looks very good, but I have some thoughts about how we might do this differently for  pop-up windows in the future. I'll send a PM.

Approved with thanks,

Chris

**Pull request merged**